Automattic/vip-design-system
# Fix new Select on Firefox

## Description

Fixed new Select on Firefox. Thanks for reporting it, @andrea-sdl and @djalmaaraujo helping to fix it

<img width="719" alt="image" src="https://user-images.githubusercontent.com/199867/193330650-e21cbea3-6ae9-48a5-94eb-143797255aa0.png">

## Checklist


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Pull down PR.
1. `npm run dev`.
1. Open Storybook on Firefox. Check if the new Select appears ok.
